### ISSUE TITLE
Switching from fragment to chatActivity, chatActionBar not found

### ISSUE THREAD

**Issue by CoAlert19** [chat-sdk/chat-sdk-android#636]
1. **Is the bug present in the demo Chat SDK project?**

2. **What modifications have you made to the Chat SDK?**
No Modification made

3. **Android Version:**
9
4. **Steps taken to reproduce the problem:**
 startActivity(new Intent(getContext(), ChatActivity.class));
        getActivity().finish();

5. **Expected result:**
Start Chat Activity

6. **Actual result:**
Caused by: java.lang.IllegalStateException: Required view 'chatActionBar' with ID 2131361972 for field 'chatActionBar' was not found. If this view is optional add '@Nullable' (fields) or '@Optional' (methods) annotation.
        at butterknife.internal.Utils.findRequiredView(Utils.java:84)
        at butterknife.internal.Utils.findRequiredViewAsType(Utils.java:96)
        at sdk.chat.ui.activities.ChatActivity_ViewBinding.<init>(ChatActivity_ViewBinding.java:33)
        at java.lang.reflect.Constructor.newInstance0(Native Method)
        at java.lang.reflect.Constructor.newInstance(Constructor.java:343)
        at butterknife.ButterKnife.bind(ButterKnife.java:170)
        at butterknife.ButterKnife.bind(ButterKnife.java:99)
        at sdk.chat.ui.activities.BaseActivity.onCreate(BaseActivity.java:90)
        at sdk.chat.ui.activities.ChatActivity.onCreate(ChatActivity.java:104)

7. **Comments:**
I have my own Navigation View in my app. I want to add chat module in my app. For this, I am switching from my fragment to ChatActivity of ChatSDK. It is giving error of chatActionBar'

**Comment by bensmiley** [comment]
@CoAlert19 this seems to happen when you have a layout file with the same name as one of the Chat SDK layout files. Then this one is used instead and it doesn't have the correct fields for ButterKnife so there is a null pointer exception. Can you confirm that this isn't the case?

**Comment by CoAlert19** [comment]
There was a similar layout file which i renamed. Now, I am getting this error.   

 java.lang.RuntimeException: Unable to start activity ComponentInfo{com.archestro.tiramap/sdk.chat.ui.activities.ChatActivity}: android.view.InflateException: Binary XML file line #14: Binary XML file line #14: Error inflating class sdk.chat.ui.appbar.ChatActionBar

**Comment by bensmiley** [comment]
@CoAlert19 did you watch the setup video? These kinds of issues are usually related to a problem with the configuration. It could be that your class doesn't inherit from the ChatSDK theme. I can also be related to your project not having the right Gradle dependencies. It's really hard for me to debug remotely because it could be related to so many things. Can you also confirm your Android version? And show me how you have imported the Chat SDK in Gradle.

**Comment by CoAlert19** [comment]
<img width="289" alt="gradle" src="https://user-images.githubusercontent.com/62545378/86516018-c7ee0e00-be36-11ea-9f0d-48e7ed791098.PNG">
![image](https://user-images.githubusercontent.com/62545378/86516034-e5bb7300-be36-11ea-9065-81e06f8bdaf6.png)
![image](https://user-images.githubusercontent.com/62545378/86516044-f79d1600-be36-11ea-825a-ac36a3c31a4e.png)

**Comment by CoAlert19** [comment]
Android Version is 9, Yes I followed the video step by step. Dependencies are not a problem. I can confirm that. It is most probably related to Chat Sdk theme. When I run it as a seperate entity, it works but when i integrate it into my app with Navigation Drawer already having its own theme, it crashes. I also tried changing theme to Chat Sdk theme in this current fragment's layout but problem persists.
